### PR TITLE
Renames the "Infrastructure & Developer Tools" to

### PR DESCRIPTION
**Pull request by annegentle**
"Infrastructure & Developer Services"

Adds copy for new section, Software Development Kits & Tools.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -292,7 +292,7 @@
...
+       <p>This collection of powerful services offers multiple utilities that work well with Rackspace Cloud resources. Cloud Identity is your starting point for accessing any cloud resource. Use Autoscale to provide automatic scaling based on Cloud Monitoring information, in response to in increase or decrease of overall workload, defined by your policies. Cloud Orchestration provides templates for building entire applications that are highly available and scalable. Use Cloud Queues as a notifications service, with basic components offering the flexibility to use different messaging patterns. Mailgun provides your application access to send or receive email, either through SMTP or the Mailgun API, while verifying the domain and helping to protect email reputation.</p>
```
Comment (annegentle) [inline index.rst]: I dunno why this gets a red/green diff but I didn't change any copy here.

Comment (smashwilson) [inline index.rst]: Whitespace changes, most likely.

Fun GitHub trick: you can check by adding `?w=1` to the end of the URL :sparkles:

Comment (smashwilson) [inline index.rst]: Hmm, that doesn't actually seem to be it. I'm not sure what changed, then.

Comment (annegentle) [inline index.rst]: I changed line 294 then changed it back so I think I confused git. :)

**Pull request merged**